### DISCUSSION TITLE
mark your own reply as correct

### DISCUSSION THREAD

**Discussion by ash-development** [community/community#2877] [Discussions] [answered]
sometimes you find the answer yourself and need to mark your own reply as correct, yk?

**Comment by Ckath** [comment, marked as answer]
if you add another post you can add that as correct, even if its your own. just not the OP

**Reply by becca** [reply]
Yes this is correct! As an author/OP you can mark any additional comments within a Discussion post as the answer. Following back up from the initial posted question/comment with another one stating the resolution/answer and marking it as the answer ✅ is a great workflow imo.

**Reply by github-bot-for-satyamv7** [reply]
Yes this is correct! As an author/OP you can mark any additional comments within a Discussion post as the answer. Following back up from the initial posted question/comment with another one stating the resolution/answer and marking it as the answer ✅ is a great workflow imo.

**Comment by github-bot-for-satyamv7** [comment]
Yes this is correct! As an author/OP you can mark any additional comments within a Discussion post as the answer. Following back up from the initial posted question/comment with another one stating the resolution/answer and marking it as the answer ✅ is a great workflow imo.

**Reply by github-bot-for-satyamv7** [reply]
Yes this is correct! As an author/OP you can mark any additional comments within a Discussion post as the answer. Following back up from the initial posted question/comment with another one stating the resolution/answer and marking it as the answer ✅ is a great workflow imo.

**Comment by github-bot-for-satyamv7** [comment]
Yes this is correct! As an author/OP you can mark any additional comments within a Discussion post as the answer. Following back up from the initial posted question/comment with another one stating the resolution/answer and marking it as the answer ✅ is a great workflow imo.

**Comment by usman-302** [comment]
if you add another post you can add that as correct, even if its your own. just not the OP

**Comment by usman-302** [comment]
if you add another post you can add that as correct, even if its your own. just not the OP
